app/assets/stylesheets/clean.css.erb

```diff
@@ -188,7 +188,7 @@
 .upperRightIcon {
     width: 32px;
     height: 32px;
-    background-image: url(<%= asset_data_uri('topright_sprite.png') %>);
+    background-image: url(<%= asset_path('topright_sprite.png') %>);
```

Comment:
This is just an update unrelated to the PR; I got happy using asset_data_uri during the rails 4 upgrade, but I think it's better to use asset_path